NumLib/Fem/CoordinatesMapping/NaturalCoordinatesMapping.h

```diff
@@ -23,6 +23,11 @@ namespace NumLib
 /**
  * Coordinates mapping tools for natural coordinates
  *
+ * This class also supports coordinates mapping of mixed dimensional elements,
+ * e.g. line elements in 2D space. Details of the mapping method can be found in
+ * Kolditz O (2001) Non-linear flow in fractured rock. International Journal of
+ * Numerical Methods for Heat & Fluid Flow 11:547–575.
+ *
```

Comment:
Please add this to doxygen references like [here](https://github.com/ufz/ogs/blob/master/GeoLib/AnalyticalGeometry.h#L59) and add the bibtex entry to [Documentation/bibliography.bib](https://github.com/ufz/ogs/blob/master/Documentation/bibliography.bib).